editor/store/selectors.js

```diff
@@ -1435,3 +1435,37 @@ export function isPublishingPost( state ) {
 export function getProvisionalBlockUID( state ) {
 	return state.provisionalBlockUID;
 }
+
+/**
+ * Returns whether the permalink is editable or not.
+ *
+ * @param {Object} state Editor state.
+ *
+ * @return {boolean} Whether or not the permalink is editable.
+ */
+export function isPermalinkEditable( state ) {
+	const samplePermalinkData = getEditedPostAttribute( state, 'sample_permalink' );
+
+	if ( ! samplePermalinkData || samplePermalinkData.length < 2 ) {
```

Comment:
When can we expect this condition to be met?

**Edit:** I'm assuming based on some of the early returns in `gutenberg_add_sample_permalink_to_draft_posts`. Would those ever be the case for the editor?

Reply:
Yah, we don't need this check anymore, it's a leftover from old behaviour.